src/route/game/index.tsx

```diff
@@ -43,6 +61,43 @@ const GameWindow: React.FC = () => {
 		};
 	}, [questionId]);
 
+	useEffect(() => {
+		switchFiftyHint(false);
+		switchCallHint('');
+	}, [questionNumber, switchCallHint, switchFiftyHint]);
+
+	const handleClickAvailableHints = useCallback(
+		(name: HintsType) => {
+			switch (name as HintsType) {
+				case HintsType.fiftyAvailable:
+					switchFiftyHint(true);
+					break;
+				case HintsType.callAvailable:
+					switchCallHint(getCallHint(questionId, fiftyHint));
+					break;
+				case HintsType.viewersAvailable:
+					setPaused(true);
+					switchViewerHint(getViewerHint(questionId, difficult, fiftyHint));
+					setTimeout(() => {
```

Comment:
Многие забывают про аргументы `setTimeout`, что второй и последующие аргументы отправляются в саму функцию:
```suggestion
					setTimeout(setPaused, 2000, false);
```